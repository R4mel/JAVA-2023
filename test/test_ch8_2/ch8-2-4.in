1
17
6
4
5
7
0
1
2
8
0
5
t33.txt
9
1
-1
9
-1
3
0
d2
3
2
t00.txt
2
0
2
1
6
0
0
0
