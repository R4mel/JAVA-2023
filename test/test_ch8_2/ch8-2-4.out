
************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 17

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 4
[0] t0.txt               F 2023-11-15 07:13        51
[1] t1.txt               F 2023-11-15 07:13       102
[2] t2.txt               F 2023-11-15 07:13       153
[3] t3.txt               F 2023-11-15 07:13       204
-----------------
[data] directory: 4 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 5
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
-----------------
[data] directory: 6 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 7
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
-----------------
[data] directory: 6 files
index number of a file to display [-1:stop]? 0
can't select directory: d0
index number of a file to display [-1:stop]? 1
can't select directory: d1
index number of a file to display [-1:stop]? 2
This is a text file to test File Management Menu.

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 8
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
-----------------
[data] directory: 6 files
index number of a source file to copy [-1:stop]? 0
can't select directory: d0
index number of a source file to copy [-1:stop]? 5
target file name? t33.txt
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 9
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files
index number of a source file to append [-1:stop]? 1
can't select directory: d1
index number of a source file to append [-1:stop]? -1

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 9
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files
index number of a source file to append [-1:stop]? -1

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 3
[0] d0                   D 2023-11-15 07:13         0
[1] d1                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files
index number of a source file to rename [-1:stop]? 0
target file name? d2
[0] d1                   D 2023-11-15 07:13         0
[1] d2                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 3
[0] d1                   D 2023-11-15 07:13         0
[1] d2                   D 2023-11-15 07:13         0
[2] t0.txt               F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files
index number of a source file to rename [-1:stop]? 2
target file name? t00.txt
[0] d1                   D 2023-11-15 07:13         0
[1] d2                   D 2023-11-15 07:13         0
[2] t00.txt              F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 2
[0] d1                   D 2023-11-15 07:13         0
[1] d2                   D 2023-11-15 07:13         0
[2] t00.txt              F 2023-11-15 07:13        51
[3] t1.txt               F 2023-11-15 07:13       102
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 7 files
index number of a file to delete [-1:stop]? 0
[0] d2                   D 2023-11-15 07:13         0
[1] t00.txt              F 2023-11-15 07:13        51
[2] t1.txt               F 2023-11-15 07:13       102
[3] t2.txt               F 2023-11-15 07:13       153
[4] t3.txt               F 2023-11-15 07:13       204
[5] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 6 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 2
[0] d2                   D 2023-11-15 07:13         0
[1] t00.txt              F 2023-11-15 07:13        51
[2] t1.txt               F 2023-11-15 07:13       102
[3] t2.txt               F 2023-11-15 07:13       153
[4] t3.txt               F 2023-11-15 07:13       204
[5] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 6 files
index number of a file to delete [-1:stop]? 1
[0] d2                   D 2023-11-15 07:13         0
[1] t1.txt               F 2023-11-15 07:13       102
[2] t2.txt               F 2023-11-15 07:13       153
[3] t3.txt               F 2023-11-15 07:13       204
[4] t33.txt              F 2023-11-15 07:13       204
-----------------
[data] directory: 5 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 0

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 0

************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 0

Good bye!!
